intellij-plugin/build.gradle.kts

```diff
@@ -148,3 +149,47 @@ tasks.register("downloadMockJdk") {
 tasks.named("test").configure {
   dependsOn("downloadMockJdk")
 }
+
+tasks.register("generateUpdatePluginsXml") {
```

Comment:
Are tasks created like this cacheable by default?

Reply:
Woops didn't think about the cache, good question, I'll 👀

Reply:
Nope, you must [ask for it explicitly](https://docs.gradle.org/current/userguide/build_cache.html#using_the_runtime_api), TIL!